magento/magento2
# Prepare jquery mobile work with new version jquery

Fix Uncaught TypeError: Cannot read property 'concat' of undefined in jquery.mobile.custom in jquery 3.6.0

1. Additional also reduce define deps jquery.mobile.custom in requirejs for frontend scope. We already defined it once as dependency at file lib/mage/menu.js
2. Remove jquery.mobile.custom from code base due update new jquery version in future. We eliminated jquery mobile for loose coupling with this oldy friend 

### Description (*)


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. This error with jquery.mobile.custom when we use higher version of jquery.
2. Open browser to see error in console tab
3. Make sure in mobile web still work without any issue

### Questions or comments

@sivaschenko @DanielRuf  you may interesting with such changes
Feel free to give me any feedbacks

https://jquerymobile.com/download/
Current jquery mobile latest version stable is 1.4.5. Magento used custom version at 1.4.3. Should we keep use this widget for next version ?


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#33312: Prepare jquery mobile work with new version jquery
